openedx_webhooks/lib/edx_repo_tools_data/test/models/test_people.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from openedx_webhooks.lib.exceptions import NotFoundError
+
+
+class TestGet:
```

Comment:
Usually once people have drunk the pytest kool-aid with fixtures, etc, they no longer write test classes.  Why do you prefer to?